shell/components/CruResource.vue

```diff
@@ -357,6 +357,7 @@ export default {
 
 <template>
   <section class="cru">
+    test
```

Comment:
@bisht-richa I don't think this should be here. You should self-review PRs once created just to check for this kind of thing.